spec/lib/efile/az/az140_calculator_spec.rb

```diff
@@ -416,6 +416,30 @@
     end
   end
 
+  describe "Line 55: Extension Payments" do
+    context "when there are no extension payments" do
+      before do
+        allow(intake).to receive(:extension_payments_amount).and_return 0
+      end
+
+      it "returns nil" do
+        instance.calculate
+        expect(instance.lines[:AZ140_LINE_55].value).to eq(0)
+      end
+    end
+
+    context "when there are extension payments" do
+      before do
+        allow(intake).to receive(:extension_payments_amount).and_return 2112
+      end
+
+      it "returns the amount of the payment" do
+        instance.calculate
+        expect(instance.lines[:AZ140_LINE_55].value).to eq(2112)
+      end
+    end
+  end
+
```

Comment:
could you also include some tests on how this impacts line 59?

Reply:
good idea